src/backend/TrafficCourts/Citizen.Service/Services/Impl/CitizenDocumentService.cs

```diff
@@ -75,6 +81,12 @@ public async Task<Coms.Client.File> GetFileAsync(Guid fileId, CancellationToken
 
         var properties = new DocumentProperties(file.Metadata, file.Tags);
 
+        // Citizen Portal should only have access to Citizen documents
+        if (properties.DocumentSource is not null && properties.DocumentSource is not DocumentSource.Citizen) {

```

Comment:
Isn't it should be Staff ? 😂

Reply:
When a user uploads a new document, documentSource is null so I need this expression.

No, I do mean Citizen here. When the retrieved document is not of type Citizen on the citizen-portal, it should never have been accessed in the first place.

If not citizen doc, throw an error.